libraries/gpu/src/gpu/State.h

```diff
@@ -134,19 +135,27 @@ class State {
         ComparisonFunction getFunction() const { return function; }
         uint8 getWriteMask() const { return writeMask; }
 
-        int32 getRaw() const { return *(reinterpret_cast<const int32*>(this)); }
-        DepthTest(int32 raw) { *(reinterpret_cast<int32*>(this)) = raw; }
-        bool operator==(const DepthTest& right) const { return getRaw() == right.getRaw(); }
-        bool operator!=(const DepthTest& right) const { return getRaw() != right.getRaw(); }
-    };
+        bool operator==(const DepthTest& right) const {
+            return
+                    writeMask == right.writeMask &&
+                    enabled   == right.enabled &&
```

Comment:
alignment looks nice, but unfortunately not following the code style of the rest of the project